nvflare/private/fed/server/training_cmds.py

```diff
@@ -52,14 +52,14 @@ def get_spec(self):
                     visible=True,
                     confirm="auth",
                 ),
-                CommandSpec(
-                    name=AdminCommandNames.DEPLOY_APP,
-                    description="deploy FL app to client/server",
-                    usage="deploy_app run_number app server|client <client-name>|all",
-                    handler_func=self.deploy_app,
-                    authz_func=self.authorize_deploy_app,
-                    visible=True,
-                ),
+                # CommandSpec(
+                #     name=AdminCommandNames.DEPLOY_APP,
+                #     description="deploy FL app to client/server",
+                #     usage="deploy_app run_number app server|client <client-name>|all",
+                #     handler_func=self.deploy_app,
+                #     authz_func=self.authorize_deploy_app,
+                #     visible=True,
+                # ),
```

Comment:
We can just delete this along with the function associated with it.
Since the old codes are in main branch.

Reply:
changed to make the deploy_app command invisible.